pkg/resource/plugin/plugin.go

```diff
@@ -120,7 +120,6 @@ func newPlugin(ctx *Context, bin string, prefix string, args []string) (*plugin,
 				break
 			}
 
-			msg = strings.TrimRightFunc(msg, unicode.IsSpace)
 			if strings.TrimSpace(msg) != "" {
```

Comment:
Note: this line is still a bit suspect.  Effectively, if we have a blank line in the output we don't bother sending it along.  Doesn't seem sensible.  But i don't wan't to touch it because it might degrade something we care about.